applications/ConvectionDiffusionApplication/tests/thermo_mechanical_tests/thermo_mech_accessor/coupled_accessor_test.mdpa

```diff
@@ -0,0 +1,310 @@
+Begin Properties 0
```

Comment:
Can't we do the same with a much coarse mesh. I think that very few elements are sufficient in this case).